lib/cyclotom.gi

```diff
@@ -2005,24 +2005,24 @@ InstallGlobalFunction( DenominatorCyc, function( cyc )
 # To simplify the code using these lists, the final entry of each list is
 # fail, resp. the trivial filter IsObject.
 BindGlobal("CompareCyclotomicCollectionHelper_Semirings", MakeImmutable( [
-	PositiveIntegers, NonnegativeIntegers,
-	Integers, GaussianIntegers,
-	Rationals, GaussianRationals,
-	Cyclotomics, fail
+        PositiveIntegers, NonnegativeIntegers,
```

Comment:
I think in this file the tabs are due to me and were meant to be 4 wide. But not important